spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/instrument/web/TraceFilter.java

```diff
@@ -98,14 +94,13 @@ protected void doFilterInternal(HttpServletRequest request,
 		Trace trace = (Trace) request.getAttribute(TRACE_REQUEST_ATTR);
 		if (trace != null) {
 			this.traceManager.continueSpan(trace.getSpan());
-		}
-		else if (skip) {
+		} else if (skip) {
 			addToResponseIfNotPresent(response, Trace.NOT_SAMPLED_NAME, "");
 		}
 
 		String spanId = getHeader(request, response, Trace.SPAN_ID_NAME);
 		String traceId = getHeader(request, response, Trace.TRACE_ID_NAME);
-		String name = "http" + uri;
+		String name = HeaderKeys.HTTP + request.getMethod();
```

Comment:
this looks like a behavior change? make sure you call things out like this so people don't miss them. Ex. often I'll self-comment on my files.